composer.json

```diff
@@ -12,16 +12,16 @@
         }
     ],
     "require": {
-        "php": ">=5.3.3",
+        "php": ">=5.5.0",
```

Comment:
For now, we need to keep the compatibility to 5.3, some projects would like to benefit to this PR. We won't support it anymore when we'll require Symfony ~3.0.
Can you re-apply the compatibility to the whole PR plz?